hummingbot/core/api_throttler/api_throttler_base.py

```diff
@@ -31,11 +32,17 @@ def __init__(self,
             for limit in self._rate_limit_list
         }
 
+        # Each path url has its own queue.
+        # Rate Limits have a DEFAULT_PATH. See data_types.py
+        self._path_task_logs_map: Dict[RequestPath, Deque[TaskLog]] = {
```

Comment:
I think it could be a good idea to associate the queues to each RateLimit directly, in case in the future we need to change them to apply for more than one URL